<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Crear Organigrama</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#27263c;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="qgv">
      <property name="minimumSize">
       <size>
        <width>850</width>
        <height>400</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(170, 170, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="crear_organigrama">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>199</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>7</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background:rgb(170, 170, 255);
</string>
         </property>
         <property name="text">
          <string>Crear organigrama</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons8-estructura-de-árbol-50.png</normaloff>icons8-estructura-de-árbol-50.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="abrir_organigrama">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>7</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background:rgb(170, 170, 255);</string>
         </property>
         <property name="text">
          <string>Abrir organigrama</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons8-abrir-carpeta-50.png</normaloff>icons8-abrir-carpeta-50.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="crear_dependencia">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>7</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background:rgb(170, 170, 255);</string>
         </property>
         <property name="text">
          <string>Crear dependencia</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons8-computadoras-conectando-30.png</normaloff>icons8-computadoras-conectando-30.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Add_Persona">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background:rgb(170, 170, 255);
</string>
         </property>
         <property name="text">
          <string>Agregar Persona</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="conectar_dependencia">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>7</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background:rgb(170, 170, 255);</string>
         </property>
         <property name="text">
          <string>Conectar dependencias</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons8-conectar-50.png</normaloff>icons8-conectar-50.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="eliminar_dependencia">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>7</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background:rgb(170, 170, 255);</string>
         </property>
         <property name="text">
          <string>Eliminar dependencia</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons8-eliminar-24.png</normaloff>icons8-eliminar-24.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
